source/Government.h

```diff
@@ -118,6 +118,9 @@ class Government {
 
 	bool IsProvokedOnScan() const;
 
+	// Get if ships from this government can go to that system.
+	bool Restricted(const System *system) const;
```

Comment:
The given system should not be nullable:
```suggestion
	bool Restricted(const System &system) const;
```

Reply:
Indeed you are right it was better if this way was to be kept.
Now I made it so you specify a system and/or a planet, meaning it needs to be nullable.